css-four-column-layout/styles.css

```diff
@@ -0,0 +1,56 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Barlow', sans-serif;
+}
+
+.container {
+  background-color: rgb(247, 248, 250);
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+}
+
+h1 {
+  text-align: center;
+}
+
+h3 {
+  text-align: center;
+}
+
+p {
+  text-align: center;
+  font-size: 15px;
+}
+
+.column-full {
+  width: 100%;
+}
+
+.column-one-fourth {
+  width: 25%;
+}
+
+.row {
+  display: flex;
+}
+
+.section {
+  display: flex;
+}
+
+.screen-img {
+  width: 100%;
+}
+
+.icon-img {
+  width: 40%;
+}
+
+.icon-img-align {
+  padding-left: 130px;
```

Comment:
You shouldn't use padding to center items. You should be able to center this image by putting it into a `div` and applying `text-align: center` to that `div`